Dockerfile/electrs/Dockerfile

```diff
@@ -35,9 +35,9 @@ RUN apt-get update -y \
 
 COPY --from=builder /usr/local/cargo/bin/electrs /usr/local/bin
 
-ENV BITCOIN_DATA=/var/lib/bitcoin
+ENV BITCOIN_DATA=/var/lib/bitcoind
```

Comment:
This will require changes to all docker-compose files using electrs, including mainnet. Otherwise they will break

Reply:
as you can see from this grep, both paths (`/var/lib/bitcoin` and `/var/lib/bitcoind`) are used:
```
git grep "/var/lib/bitcoin"
Dockerfile/bitcoind-signet/Dockerfile:VOLUME "/var/lib/bitcoin"
Dockerfile/bitcoind-signet/Dockerfile:ENV BITCOIN_DATA=/var/lib/bitcoin
Dockerfile/bitcoind/Dockerfile:VOLUME "/var/lib/bitcoind"
Dockerfile/bitcoind/Dockerfile:ENV BITCOIN_DATA=/var/lib/bitcoind
Dockerfile/bitcoind/Dockerfile:ENTRYPOINT ["bitcoind", "-txindex", "-datadir=/var/lib/bitcoind"]
Dockerfile/bitcoind/with-wallet/Dockerfile:VOLUME "/var/lib/bitcoind"
Dockerfile/bitcoind/with-wallet/Dockerfile:ENV BITCOIN_DATA=/var/lib/bitcoind
Dockerfile/electrs-signet/Dockerfile:ENV BITCOIN_DATA=/var/lib/bitcoin
Dockerfile/electrs-signet/Dockerfile:VOLUME "/var/lib/bitcoind"
Dockerfile/electrs/Dockerfile:ENV BITCOIN_DATA=/var/lib/bitcoind
Dockerfile/electrs/Dockerfile:VOLUME "/var/lib/bitcoind"
README.md:docker volume create --driver local --opt o=bind --opt type=none --opt device=/var/lib/bitcoin bitcoin 
README.md:      - "blockchain:/var/lib/bitcoind"
README.md:      - "/Volumes/ExternalDrive/bitcoin:/var/lib/bitcoind"
README.md:  -v /Volumes/ExternalDrive/bitcoin:/var/lib/bitcoind \
docker-compose/mainnet/docker-compose.yml:      - bitcoin:/var/lib/bitcoind
docker-compose/mainnet/docker-compose.yml:        target: /var/lib/bitcoin
docker-compose/signet/docker-compose.yml:      - bitcoin:/var/lib/bitcoin
docker-compose/signet/docker-compose.yml:        target: /var/lib/bitcoin
docker-compose/testnet/docker-compose.yml:      - bitcoin:/var/lib/bitcoind
docker-compose/testnet/docker-compose.yml:        target: /var/lib/bitcoind
```

they all should be the same, so tell me which path do you prefer and I'll update this PR

Reply:
Yeah, there is inconsistency here, but I was not proofing it not to break other things. This change breaks electrs image for current mainnet compose file and will require to change both mainnet and signet compose files + electrs Dockerfile for signet.

Reply:
ok let's stick with `/var/lib/bitcoin` so we don't need to change mainnet compose and electrs dockerfile, I'll fix the path in the testnet compose and everywhere I see `/var/lib/bitcoind`

Reply:
The point is that both mainnet and testned are using the same Dockerfiles, meaning that the path must be the same everywhere...